### ISSUE TITLE
Pass eslint

### ISSUE THREAD

**Issue by a-unique-id** [TeamShellSMM/ShellBot3000#56]
When you run `npm run lint`, we currently have a lot of errors and warnings. When we get that to 0, we can add it to the checks and tests we do automatically.

This might be a good place to talk about some eslint rules that we want or don't want too before we finalize it.

**Comment by a-unique-id** [comment]
One of the more problematic rules is the "require camelCase" one. Our database tables mostly use snake_case for names as a convention, and we don't use Objection.js to the fullest to map the properies from snake_case to camelCase. We also have some raw MySQL commands too currently  so not sure how easy it is to move it to Objections.js again

I think there's an option to be more lenient and allow snake_case on properties but we ended up using a lot of destructuring too, which might still trigger it

**Comment by a-unique-id** [comment]
Resolved with c9b3c725e6c68e899c0a87d4ed3b66a96d5c23e0 and added to travis.yml in a7cc4cd4f2888d058a65a8bbbf5c9ccfa5b7decb

We have added some exceptions in the current eslint that in the future we'll remove. We've allowed two variables for camelcase `"guild_id", "discord_id"`.

We removed the checks `for...of` from the `"no-restricted-syntax"` rule and also `"no-await-in-loop"` as we do a lot of processing of async database calls in loops that needs to be synchronous. 

As we work on the code, we can revisit our style guidelines but for now we will consider this closed
